src/UserList.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import usersData from './data/users.json';
+import User from './components/user.js';
+import './css/App.css';
+
+
+export default class UserList extends React.Component {
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            wantAdd: false
+        };
+
+        this.handleClickAddUser = this.handleClickAddUser.bind(this);
+    }
+
+    handleClickAddUser () {
+        this.setState({ wantAdd: !this.state.wantAdd });
+    }
+
+    render () {
+        const { wantAdd } = this.state;
+        return (
+            <div className="App">
```

Comment:
It's better when the top css class of an element is the same (or at least similar) with its class name